### PR TITLE
Fixed readme for validate

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

### CODE REVIEW

**Comment by Itay4** [comment]
it looks like we have regression here, we should have a flag for validating id_set, and it was replaced with input flag
lets add a flag for it

**Comment by Shellyber** [comment]
@Itay4 Please have a look

File: demisto_sdk/commands/validate/README.md [hunk 1]
```diff
@@ -48,8 +48,8 @@ This will validate only changed files from content origin/master branch and will
 compatibility checks.
 <br><br>
 
-`demisto-sdk validate -i -j`
-This will validate all content repo files and including conf.json file and will create the id_set.json file.
+`demisto-sdk validate -j`
+This will validate all content repo files and including conf.json file.
```
Comment (Itay4) [inline demisto_sdk/commands/validate/README.md:52]: i think this validates only conf json, not all files

File: demisto_sdk/commands/validate/README.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ Path of file to validate specifically.
 * **-a, --validate-all**
 Whether to run all validation on all files or not.
 * **-i, --input**
-The path of a pack to validate specifically.
+The path of a pack or a file to validate specifically.
 
 **Examples**:
 `demisto-sdk validate`
```
Comment (Itay4) [inline demisto_sdk/commands/validate/README.md:42]: lets remove this example

File: demisto_sdk/commands/validate/README.md (hunk 1 above)
Comment (Shellyber) [reply demisto_sdk/commands/validate/README.md:52]: No, definitely all files. you can test it.
That's what we talked about yesterday

**Pull request merged**